pycoshark/mongomodels.py

```diff
@@ -498,7 +498,25 @@ class ActionWorkflow(Document):
     updated_at = DateTimeField()
     project_url = StringField(default=None)
     project_id = ObjectIdField(default=None)
-    vcs_system_id = ObjectIdField(default=None)
+    project = ObjectIdField(default=None)
```

Comment:
For what do we use project that could not be done with project_id?

Reply:
oh yes it's redundant and we only use project_id. I will remove it.